src/Sql/dbo/Tables/Project.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[Project] (
+    [Id]                UNIQUEIDENTIFIER NOT NULL,
+    [OrganizationId]    UNIQUEIDENTIFIER NOT NULL,
+    [Name]              NVARCHAR(MAX) NULL, 
+    [CreationDate]      DATETIME2 (7),
+    [RevisionDate]      DATETIME2 (7), 
+    [DeletedDate]       DATETIME2 (7) NULL,
+    CONSTRAINT [PK_Project] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_Project_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id]),
```

Comment:
We will want this to `ON DELETE CASCADE` for the Organization FK

Reply:
Thank you! Updated